requires.py

```diff
@@ -45,3 +46,12 @@ def get_private_ip(self):
     def get_rest_port(self):
         conv = self.conversation()
         return conv.get_remote('rest_port')
+
+    def get_master_info(self):
+        conv = self.conversation()
+        data={
+            'connection_string': conv.get_remote('connection_string'),
+            'master': conv.get_remote('master'),
+        }
+        return data
+
```

Comment:
This blank line at the end makes the linter fail, when run via "make lint" on the built charm. At least it's an easy fix :-)